components/AdminPanel/AdminPanel.tsx

```diff
@@ -1,3 +1,83 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { useCallback } from 'react';
+
+import { useQuery } from '../../hooks/useQuery';
+import { getBlogs } from '../../utils/api/getBlogs';
+import { formatDate } from '../../utils/date-utils';
+import { addTrackingToLink } from '../../utils/link-utils';
+import { BlogsTable } from '../BlogsTable/BlogsTable';
+
+import styles from './adminPanel.module.scss';
+
 export const AdminPanel = () => {
-  return <section>admin panel</section>;
+  const router = useRouter();
```

Comment:
```suggestion
  const { push } = useRouter();
```

Reply:
Moim zdaniem `router.push` powinno zostać, bo wiadomo do czego się coś pushuje i co to zrobi.